test/e2e/apps/disruption.go

```diff
@@ -474,7 +474,7 @@ var _ = SIGDescribe("DisruptionController", func() {
 			rs.Labels["name"] = rsName
 			initialDelaySeconds := framework.PodStartTimeout.Seconds() + 30
 			if tc.podsShouldBecomeReadyFirst {
-				initialDelaySeconds = 0
+				initialDelaySeconds = 20
```

Comment:
Originally I set a 0 here to limit the run time of the test. But we need some readiness delay to prevent the raciness with the replicaset and disruption controller.